src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -98,10 +98,11 @@ public function load(string $path, string ...$extraPaths): void
      * .env.local is always ignored in test env because tests should produce the same results for everyone.
      * .env.dist is loaded when it exists and .env is not found.
      *
-     * @param string      $path       A file to load
-     * @param string|null $envKey     The name of the env vars that defines the app env
-     * @param string      $defaultEnv The app env to use when none is defined
-     * @param array       $testEnvs   A list of app envs for which .env.local should be ignored
+     * @param string      $path                 A file to load
+     * @param string|null $envKey               The name of the env vars that defines the app env
+     * @param string      $defaultEnv           The app env to use when none is defined
+     * @param array       $testEnvs             A list of app envs for which .env.local should be ignored
+     * @param bool        $overrideExistingVars Whether existing environment variables set by the system must be overridden (@see https://symfony.com/doc/current/configuration.html#overriding-environment-variables-defined-by-the-system)
```

Comment:
we try to always use "should" instead of "must" ;) + no doc link.
I'm going to update while merging

Reply:
Alright, I'll think of this in the next occasion. Thank you for taking care of this!